Jenkinsfile

```diff
@@ -77,16 +77,18 @@ def cleanup_master() {
 }
 
 def archive_logs() {
-	sh """
-	cd "\$(echo ${env.WORKSPACE} | cut -f 1 -d '@')"
-	LOGS_NAME=\$(echo "logs-$JOB_NAME-$NODE_NAME-$BUILD_ID" | sed 's/\\///g')
-	mv logs "\$LOGS_NAME"
+	sh '''
+	cd "$(echo "$WORKSPACE" | cut -f 1 -d '@')"
+	# need a unique and descriptive name so that the artifacts don't clobber each other
+	# and we can know which log file goes to which node, etc
+	LOGS_NAME=$(echo "logs-$JOB_NAME-$NODE_NAME-$BUILD_ID" | sed 's/\\///g')
```

Comment:
I don't think that sed expression is going to work. What is it supposed to do?

Reply:
remove the "/" that otherwise shows up in LOGS_NAME

Reply:
```
~> echo 'foo/' | sed 's/\\///g'
sed: 1: "s/\\///g": bad flag in substitute command: '/'
```